packages/react/src/components/TableOfContents/__stories__/TableOfContents.stories.js

```diff
@@ -46,124 +44,105 @@ if (DDS_TOC) {
-              consectetur. Praesent fermentum sodales facilisis. Mauris a
-              efficitur sem. Aliquam vehicula sapien libero, a viverra felis
-              scelerisque vel. Vestibulum ante ipsum primis in faucibus orci
-              luctus et ultrices posuere cubilia Curae; Donec fringilla dui
-              tellus, a pretium diam vehicula et. Etiam non vulputate augue.
-              Morbi laoreet diam dapibus sapien pellentesque tristique. Morbi id
-              nibh metus. Integer non scelerisque nisl.
-            </p>
-            <a name="3" style={{ color: '#000' }}>
-              <h3 style={{ paddingBottom: '1rem', paddingTop: '2rem' }}>
-                Morbi id nibh metus
-              </h3>
-            </a>
-            <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
-              molestie condimentum consectetur. Nulla tristique lacinia elit, at
-              elementum dui gravida non. Mauris et nisl semper, elementum quam
-              non, lacinia purus. Vivamus aliquam vitae sapien volutpat
-              efficitur. Curabitur sagittis neque facilisis magna posuere
-              consectetur. Praesent fermentum sodales facilisis. Mauris a
-              efficitur sem. Aliquam vehicula sapien libero, a viverra felis
-              scelerisque vel. Vestibulum ante ipsum primis in faucibus orci
-              luctus et ultrices posuere cubilia Curae; Donec fringilla dui
-              tellus, a pretium diam vehicula et. Etiam non vulputate augue.
-              Morbi laoreet diam dapibus sapien pellentesque tristique. Morbi id
-              nibh metus. Integer non scelerisque nisl.
-            </p>
-            <a name="14" style={{ color: '#000' }}>
-              <h3 style={{ paddingBottom: '1rem', paddingTop: '2rem' }}>
-                Integer non scelerisque
-              </h3>
-            </a>
-            <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
-              molestie condimentum consectetur. Nulla tristique lacinia elit, at
-              elementum dui gravida non. Mauris et nisl semper, elementum quam
-              non, lacinia purus. Vivamus aliquam vitae sapien volutpat
-              efficitur. Curabitur sagittis neque facilisis magna posuere
-              consectetur. Praesent fermentum sodales facilisis. Mauris a
-              efficitur sem. Aliquam vehicula sapien libero, a viverra felis
-              scelerisque vel. Vestibulum ante ipsum primis in faucibus orci
-              luctus et ultrices posuere cubilia Curae; Donec fringilla dui
-              tellus, a pretium diam vehicula et. Etiam non vulputate augue.
-              Morbi laoreet diam dapibus sapien pellentesque tristique. Morbi id
-              nibh metus. Integer non scelerisque nisl.
-            </p>
-          </TableOfContents>
-        </div>
+        <TableOfContents
+          theme={select('theme', themes, themes['light (white)'])}
```

Comment:
I changed the object key name to just `white` but forgot to change the field here. 😅 Can you change `themes['light (white)']` to `themes.light`?